src/docbkx/chapter22-improve.xml

```diff
@@ -538,7 +538,7 @@
 
 			<para>
 				Let's add an Equals filter to the job in order to write only these
-				SURVIVOR and UNIQUE records, to create a data set with has no duplicates.
+				SURVIVOR and UNIQUE records, to create a data set with no duplicates.
```

Comment:
I'm pretty sure I meant to write "_which_ has no duplicates", but this is also good.